examples/notebooks/get-started/learn_beam_transforms_by_doing.ipynb

```diff
@@ -0,0 +1,755 @@
+      "metadata": {
+        "id": "Vokbrhhyto7H"
+      },
+      "outputs": [],
+      "source": [
+        "!mkdir -p data\n",
+        "!gsutil cp gs://dataflow-samples/shakespeare/kinglear.txt data/"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "R-4uyn0Tttr2"
+      },
+      "outputs": [],
+      "source": [
+        "import re\n",
+        "\n",
+        "# Function used to run and display the result\n",
+        "def run(cmd):\n",
+        "  print('>> {}'.format(cmd))\n",
+        "  !{cmd}\n",
+        "  print('')\n",
+        "\n",
+        "inputs_pattern = 'data/*'\n",
+        "outputs_prefix = 'outputs/part'\n",
+        "\n",
+        "# Running locally in the DirectRunner.\n",
+        "with beam.Pipeline() as pipeline:\n",
+        "  word_count = (\n",
+        "      pipeline\n",
+        "        | 'Read lines' >> beam.io.ReadFromText(inputs_pattern)\n",
+        "        | 'Find words' >> beam.FlatMap(lambda line: re.findall(r\"[a-zA-Z']+\", line))\n",
+        "        | 'Pair words with 1' >> beam.Map(lambda word: (word, 1))\n",
+        "        | 'Group and sum' >> beam.CombinePerKey(sum)\n",
+        "        | 'Write results' >> beam.io.WriteToText(outputs_prefix)\n",
+        "  )\n",
+        "\n",
+        "# Sample the first 20 results, remember there are no ordering guarantees.\n",
+        "run('head -n 20 {}-00000-of-*'.format(outputs_prefix))"
+      ]
+    },
+    {
+      "attachments": {},
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "wl8wLwZZtbnX"
+      },
+      "source": [
+        "Although the code above is a viable way to design your `Pipeline`, you can see that we use multiple transforms to perform one process:\n",
+        "1. `FlatMap` is used to find words in each line\n",
+        "2. `Map` is used to create key-value pairs with each word where the value is 1\n",
+        "3. `CombinePerKey` is used so that we can then group by each word and count up the sums\n",
+        "\n",
+        "All of these `PTransforms`, in combination, are meant to count each word in *King Lear*. You can simplify the process and combine these three transforms into one by using composite transforms.\n",
+        "\n",
+        "There's two ways you can follow:\n",
+        "1. Using Beam SDK's built-in composite transforms\n",
+        "2. Creating your own composite transforms"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "nPG11AEkNMKK"
+      },
+      "source": [
+        "### 3.1 Beam SDK Composite Transforms\n",
+        "Beam makes it easy for you with its Beam SDK which comes with a package of many useful composite transforms. We will only cover one in this tutorial but to see a list of transforms you can use, see the following API reference page: [Pre-written Beam Transforms for Python](https://beam.apache.org/releases/pydoc/current/apache_beam.transforms.html).\n"
```

Comment:
```suggestion
        "Beam allows combining a sequence of transforms into a composite transform. Many of the Beam's handy pre-written transforms are composite transforms under the hood. We cover one  in this tutorial but to see other transforms you can use, check out the following API reference pages: [Beam Transforms Package](https://beam.apache.org/releases/pydoc/current/apache_beam.transforms.html), [Beam ML Package](https://beam.apache.org/releases/pydoc/current/apache_beam.ml) .\n"
```

Reply:
Polished it based on your suggestions. Thanks.